hw08_envdir_tool/go.mod

```
@@ -1,3 +1,5 @@
-module github.com/fixme_my_friend/hw08_envdir_tool
+module github.com/otus-hw-test/Ayupov-Ayaz/hw08_envdir_tool
```

```suggestion
module github.com/Ayupov-Ayaz/otus-hw-test/hw08_envdir_tool
```
Сначала профиль, затем название репозитория